src/components/ProductsBrowsing.tsx

```diff
@@ -0,0 +1,84 @@
+import { useEffect, useState } from 'react'
+import ProductCard, { ProductCardProps } from './ProductCard'
+import SortAndSearch from './SortAndSearch'
+import Filters from './Filters'
+import { exampleProducts } from '~/data/exampleProducts'
+import { Drawer } from '@mui/material'
+import useWindowDimensions from '~/hooks/useWindowDimensions'
+import CategoriesTabs from './CategoriesTabs'
+
+const ProductsBrowsing = () => {
+	const [products, setProducts] = useState<ProductCardProps[]>(exampleProducts)
+	const [searchTerm, setSearchTerm] = useState('')
+	const [filteredProducts, setFilteredProducts] = useState<ProductCardProps[]>([])
+	const [open, setOpen] = useState(false)
+	const { height, width } = useWindowDimensions()
```

Comment:
Same thing. Using the window size as a state is suboptimal. Pls use `mediaQuery`.